apps/azure-functions-api/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "azure-functions-api",
+  "version": "0.0.1",
+  "description": "Azure Function REST API Template",
+  "main": "./dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "shx rm -rf dist && npm run generate",
```

Comment:
Why?

Reply:
since with yarn hooks aren't run automatically there's no reason to follow the npm hooks naming. If we want a cleaning op, I'd call this just

`"clean": "shx rm -rf dist"`

In order to have atomic (one purpose) tasks.

Reply:
Refactored in [d112913](https://github.com/pagopa/dx-typescript/pull/4/commits/d1129132f4766532ce0d75369e6e879c723a0115)